src/client/pages/AssessmentSection/AssessmentSection.tsx

```diff
@@ -11,6 +11,7 @@ import { useIsSectionDataEmpty } from '@client/store/pages/assessmentSection/hoo
 import { useCanEditDescriptions, useCanEditTableData } from '@client/store/user/hooks'
 import { useCountryIso } from '@client/hooks'
 import { useIsPrint } from '@client/hooks/useIsPath'
+import { useListenNodeUpdates } from '@client/pages/AssessmentSection/DataTable/Table/Row/RowData/Cell/hooks/useListenNodeUpdates'
```

Comment:
I would copy that file in @client/pages/AssessmentSection and use relative import. to this and line below